### DISCUSSION TITLE
Closed-Form Interpretation of Neural Network Classifiers with Symbolic Regression Gradients

### DISCUSSION THREAD

**Discussion by sjwetzel** [MilesCranmer/PySR#520] [Show and tell]
Thank you all and especially Miles for setting up this library and all the helpful comments here! It helped me to develop a neural network interpretation method tailored to what I need for my artificial scientific discovery projects. 

https://arxiv.org/abs/2401.04978

**Comment by MilesCranmer** [comment]
Nice work, congrats!

By the way, if you want, you could add it here: https://github.com/MilesCranmer/PySR/blob/master/docs/papers.yml and it will show up on the applications page: https://astroautomata.com/PySR/papers/.

**Reply by sjwetzel** [reply]
That's a good idea. I hope I did it right, I am normally not in the business of contributing to public GitHub repos.
